BlackMarket/Views/Dashboard/PromoShipmentsView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  PromoShipments.swift
+//  BlackMarket
+//
+//  Created by Santiago Andracchi on 22/09/2023.
+//
+
+import SwiftUI
+
+struct PromoShipmentsView: View {
```

Comment:
this view is repeated in BlackMarket/Views/Dashboard/PromoShipments.swift, not sure if this is compiling given the duplicate structure name.